charts/caluma/templates/frontend-service.yaml

```diff
@@ -1,14 +1,14 @@
 apiVersion: v1
 kind: Service
 metadata:
-  name: {{ template "caluma.fullname" . }}-frontend
+  name: {{ .Values.frontend.service.name }}
```

Comment:
I don't understand why changing the generated service name to a value given service name.

Reply:
Let's ask about this change @derrabauke , can be done post PR